test/predictions/repo_test.exs

```diff
@@ -1,24 +1,41 @@
 defmodule Predictions.RepoTest do
```

Comment:
This file needs a lot of work. I removed Bypass and started implementing Mox, but realized we need to come up with a strategy for testing repos because this is the only one that used Bypass. We probably need an entire suite of Factories and I didn't want to take on that chunk of work in this PR.